flink-end-to-end-tests/test-scripts/kafka-common.sh

```diff
@@ -36,7 +36,7 @@ function setup_kafka_dist {
   mkdir -p $TEST_DATA_DIR
   KAFKA_URL="https://archive.apache.org/dist/kafka/$KAFKA_VERSION/kafka_2.11-$KAFKA_VERSION.tgz"
   echo "Downloading Kafka from $KAFKA_URL"
-  curl "$KAFKA_URL" --retry 10 --retry-max-time 120 > $TEST_DATA_DIR/kafka.tgz
+  retry_times 5 0 "curl ${KAFKA_URL} --retry 10 --retry-max-time 120 --output ${TEST_DATA_DIR}/kafka.tgz"
```

Comment:
Weren't we already retrying the curl command by setting `--retry 10`?

Reply:
Thanks for mention this. Yes, we could rely on the curl native mechanism. However, after reading the [curl manual](https://curl.haxx.se/docs/manpage.html). I found it's better to add the `--retry-delay` configure here to disable the exponential backoff mechanism. WDYT?

Reply:
En exponential backoff is not necessarily bad. We've capped it to 120s which sounds fair to me. Hence, I don't think that we have to change it.